### PR TITLE
Add "multiqueue" flag to /play, allowing users to add a full playlist to queue.

### PR DESCRIPTION
**Pull request by HelloChrisLowe**

### CODE REVIEW

**Review by MatthewMerrill — COMMENTED** [review]
nit: wdyt of "all" option name

File: src/main/java/com/mattmerr/beets/commands/PlayCommand.java [hunk 1]
```diff
@@ -56,10 +62,17 @@ public Mono<Void> execute(SlashCommandEvent event) {
     logCall(event);
     String beet = asRequiredString(event.getOption("beet"));
     Snowflake guildId = requireGuildId(event.getInteraction());
+
+    var multiqueue = asBoolean(event.getOption("multiqueue")).orElse(false);
     var delayMinutes = asLong(event.getOption("delay"));
-    if (delayMinutes.isPresent() && (delayMinutes.get() < 1 || delayMinutes.get() > 120)) {
-      return event.reply("Delay must be in range [1, 120] minutes");
+    if (delayMinutes.isPresent()) {
```
Comment (MatthewMerrill) [inline src/main/java/com/mattmerr/beets/commands/PlayCommand.java]: I sort of prefer just adding the check and repeating .isPresent() call to reduce nesting
```
if (delayMinutes.isPresent() && multiqueue) {
  return event.reply("You cannot enable multiqueue and delay at the same time");
}
if (delayMinutes.isPresent() && .....
```

File: src/main/java/com/mattmerr/beets/commands/PlayCommand.java [hunk 2]
```diff
@@ -71,9 +84,11 @@ public Mono<Void> execute(SlashCommandEvent event) {
         log.error("Error looking up beet by name", e);
       }
     }
-    AudioTrack track = beetLoader.getTrack(beet);
+
+    List<AudioTrack> tracks = beetLoader.getTracks(beet, multiqueue);
     final var beetCapture = beet;
     if (delayMinutes.isPresent()) {
+      AudioTrack track = tracks.iterator().next(); // Cannot multiqueue and delay, so this will always have 1 element.
```
Comment (MatthewMerrill) [inline src/main/java/com/mattmerr/beets/commands/PlayCommand.java]: If we're keeping GetTrack we should use that

File: src/main/java/com/mattmerr/beets/util/CachedBeetLoader.java [hunk 3]
```diff
@@ -49,8 +53,16 @@ private synchronized AudioTrack fetchTrack(String beet) {
   }
 
   public synchronized AudioTrack getTrack(String beet) {
```
Comment (MatthewMerrill) [inline src/main/java/com/mattmerr/beets/util/CachedBeetLoader.java:55]: Is this still used?

**Comment by MatthewMerrill** [comment]
LGTM, Approval once comments resolved/acked

File: src/main/java/com/mattmerr/beets/util/CachedBeetLoader.java (hunk 3 above)
Comment (HelloChrisLowe) [reply src/main/java/com/mattmerr/beets/util/CachedBeetLoader.java:55]: Yes, for clip and Discordle detection.

File: src/main/java/com/mattmerr/beets/commands/PlayCommand.java (hunk 2 above)
Comment (HelloChrisLowe) [reply src/main/java/com/mattmerr/beets/commands/PlayCommand.java]: Done.

File: src/main/java/com/mattmerr/beets/commands/PlayCommand.java (hunk 1 above)
Comment (HelloChrisLowe) [reply src/main/java/com/mattmerr/beets/commands/PlayCommand.java]: Done.

**Comment by HelloChrisLowe** [comment]
> nit: wdyt of "all" option name

Done.

**Pull request merged**